### DISCUSSION TITLE
I'm a verified student and I still can't get copilot free - Been at least a month now

### DISCUSSION THREAD

**Discussion by TheMoonlightProgrammer** [community/community#51156] [GitHub Education]
### Select Topic Area

Question

### Body

Hi, so I've been on the student plan for a while and I still can't get free access to copilot for some reason. I'm a verified student and have access to the Student Developer Pack but it still asks for some type of payment. I've tried to research if there was something else I had to do or activate but nothing has helped me or shown that I did. I know there have been similar posts and I looked at those too, but they all seemed to either resolve themselves within a few hours or have no answers. 

This is one of the many websites I used to try and figure this out, though it didn't really help that much like all the others.
https://techcommunity.microsoft.com/t5/educator-developer-blog/step-by-step-setting-up-github-student-and-github-copilot-as-an/ba-p/3736279

![image](https://user-images.githubusercontent.com/90926462/228040288-71dd6be2-bcf8-4eda-9aba-8c8d9819de0f.png)
![Screenshot 2023-03-27 135127](https://user-images.githubusercontent.com/90926462/228040058-5c432ffa-fc78-4653-96c8-84331528f099.png)

**Comment by 1205129045x** [comment]
我也是啊，你这是等了一个多月吗 我今天刚申请通过也是这样
